tensorflow/lite/kernels/add_n.cc

```diff
@@ -42,6 +58,21 @@ TfLiteStatus Prepare(TfLiteContext* context, TfLiteNode* node) {
     TF_LITE_ENSURE_EQ(context, input1->type, input->type);
   }
 
+  OpData* data = reinterpret_cast<OpData*>(node->user_data);
+  data->num_inputs = NumInputs(node);
+  if (output->type == kTfLiteFloat32) {
+    data->all_inputs = reinterpret_cast<void*>(
+        new VectorOfTensors<float>(*context, *node->inputs));
```

Comment:
Note that you didn't manage to free this piece of memory at the end. It will cause memory leaks.